sdk/test-utils/testing-recorder-new/test/noOpCredentialTest.spec.ts

```diff
@@ -45,8 +45,11 @@ describe(`NoOp credential with Tables`, () => {
       "table-name",
       `table${Math.ceil(Math.random() * 1000 + 1000)}`
     );
-    const client = new TableServiceClient(assertEnvironmentVariable("TABLES_URL"), credential);
-    recorder.configureClient(client);
+    const client = new TableServiceClient(
+      assertEnvironmentVariable("TABLES_URL"),
+      credential,
+      recorder.configureClientOptions({})
```

Comment:
nit: is it possible to also accept `undefined`? I am thinking of the case where I already have an `options?` of the original client options type and pass it here.

Reply:
Can do. 
I didn't want to do that, just to force the users to pass something.. 
to not have any confusion when they actually want to pass options.

Reply:
If you feel strongly, I can allow undefined.

Reply:
Either way is fine.